tools/harness-automation/cases/border_7_1_1.py

```diff
@@ -32,14 +32,14 @@
 import unittest
 
 
-class Leader_7_1_1(HarnessCase):
```

Comment:
In R1.4.0, the case is leader_7_1_1, right? If simply changing it to border_7_1_1, could the tool still work with R1.4.0?

Reply:
if version <= R1.4.0, have to use previous cases

Reply:
Now supports TH R 1.4.0 and higher version